lib/atom-css-sniffer.js

```diff
@@ -4,16 +4,96 @@ let helpers;
 let path;
 let filesystem;
 
+// Caches
+let versions = {};
+let cmds = {}
+
 function loadDeps() {
-  if (!helpers) {
-    helpers = require('atom-linter');
-  }
-  if (!path) {
-    path = require('path');
-  }
-  if (!filesystem) {
-    filesystem = require('fs');
-  }
+    if (!helpers) {
+        helpers = require('atom-linter');
+    }
+    if (!path) {
+        path = require('path');
+    }
+    if (!filesystem) {
+        filesystem = require('fs');
+    }
+}
+
+function _getCommand(projectPath) {
+    if (cmds[projectPath]) {
+        return cmds[projectPath];
+    }
+
+    let cmd = projectPath + '/vendor/bin/css-sniff';
+
+    if (!filesystem.existsSync(cmd)) {
+        // do we have a WWW folder?
+        cmd = projectPath + '/www/vendor/bin/css-sniff';
+
+        if (!filesystem.existsSync(cmd)) {
+            cmd = null;
+        }
+    }
+
+    return cmds[projectPath] = cmd;
+}
+
+function _checkVersion(projectPath, cmd) {
+    if (versions[projectPath]) {
+        return versions[projectPath];
+    }
+
+    return versions[projectPath] = helpers.exec(cmd, ['--version'], {}).then(function (result) {
+        let matches = result.match(/v([0-9]\.[0-9])/);
```

Comment:
Should we support `1.2.3` and `1.23` ?

Reply:
no, the version should be API version of the sniffer. so 2.0 is fine.